inc/admin/namespace.php

```diff
@@ -4,7 +4,6 @@
 
 use function HumanMade\SimpleSaml\instance;
 use function HumanMade\SimpleSaml\is_sso_enabled_network_wide;
-use OneLogin_Saml2_IdPMetadataParser;
```

Comment:
This is the opposite of what we should be doing with namespaces. Why take away the `use` statement?

Reply:
I found having the namespace in the references more readable, but that's maybe just me.

Reply:
Per [coding standards](https://new-engineering.hmn.md/standards/style/php/#namespace-and-class-naming):

> Classes used in a namespaced file should have a `use` statement at the top of the file to allow a quick overview of what is used, and to avoid long absolute references to the classes throughout the file. If you have a lot of functionality from given namespace, you can `use` the entire namespace to avoid excessive use statements.

All the classes here used within the file should be `use`'d at the start rather than absolute refs throughout.